### DISCUSSION TITLE
Add the ability to retrieve raw stats data using a stats API endpoint

### DISCUSSION THREAD

**Discussion by sunstep** [Freika/dawarich#169] [Ideas]
I would like to request raw stats data from the API, so if I want to include my Dawarich stats in one of my apps, I can easily retrieve them from an API endpoint such as /api/v1/stats. An example of the output would be:
```
{
	"TotalDistance": n,
	"GeopointsTracked": n,
	"ReverseGeocodedPoints": n,
	"CountriesVisited": n,
	"CitiesVisited": n,
	"YearlyStats": [
		{
			"Year": n,
			"TotalDistance": n,
			"CountriesVisited": n,
			"CitiesVisited": n,
			"MonthlyDistance": [
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n
			]
		},
		{
			"Year": n,
			"TotalDistance": n,
			"CountriesVisited": n,
			"CitiesVisited": n,
			"MonthlyDistance": [
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n,
				n
			]
		}
	]
}
```

This is basically just the stats page, but put in a JSON. It would be great if we could obtain this information from the API. The JSON above is checked and is valid, but it's just an example. Also, n should be numbers, obviously.

**Comment by sunstep** [comment]
In addition: it would also be great if the update stats button functionality was exposed to the API.

**Comment by Freika** [comment]
I'm closing it in favor of this thread: https://github.com/Freika/dawarich/discussions/178
